velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -191,6 +292,48 @@ std::vector<SortingKeyAndOrder> WindowFuzzer::generateSortingKeysAndOrders(
   return results;
 }
 
+template <TypeKind TKind>
+void WindowFuzzer::addOffsetColumnsToInput(
```

Comment:
nit: Since this method is specific to k-range frames, let's rename it as addKRangeOffsetColumnToInput. The original addKRangeOffsetColumnToInput can be renamed as addKRangeOffsetColumnToInputImpl.